pkg/ui/config.go

```diff
@@ -67,15 +67,15 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.Goldfish.Storage.Type, "ui.goldfish.storage.type", "", "Storage backend type (cloudsql, rds, or empty for no storage)")
 
 	// CloudSQL flags
-	f.StringVar(&cfg.Goldfish.Storage.CloudSQLUser, "ui.goldfish.cloudsql.user", "", "CloudSQL username for Goldfish database.")
-	f.StringVar(&cfg.Goldfish.Storage.CloudSQLHost, "ui.goldfish.cloudsql.host", "127.0.0.1", "CloudSQL host for Goldfish database.")
-	f.IntVar(&cfg.Goldfish.Storage.CloudSQLPort, "ui.goldfish.cloudsql.port", 3306, "CloudSQL port for Goldfish database.")
-	f.StringVar(&cfg.Goldfish.Storage.CloudSQLDatabase, "ui.goldfish.cloudsql.database", "goldfish", "CloudSQL database name for Goldfish.")
+	f.StringVar(&cfg.Goldfish.Storage.CloudSQLUser, "ui.goldfish.storage.cloudsql-user", "", "CloudSQL username for Goldfish database.")
```

Comment:
Or should we keep it as
```suggestion
	f.StringVar(&cfg.Goldfish.Storage.CloudSQLUser, "ui.goldfish.storage.cloudsql.user", "", "CloudSQL username for Goldfish database.")
```

That way we would not need to change our internal configuration.